src/EA.Weee.RequestHandlers/Mappings/UKCompetentAuthorityDataMap.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace EA.Weee.RequestHandlers.Mappings
+{
+    using Core.Shared;
+    using Domain;
+    using EA.Weee.DataAccess;
+    using Prsd.Core.Mapper;
+    using System.Linq;
+
+    public class UKCompetentAuthorityDataMap : IMap<UKCompetentAuthorityData, UKCompetentAuthority>
+    {
+        private readonly WeeeContext context;
```

Comment:
Unit Tests

Reply:
Hard to do as it needs a WeeeContext, which can't be mocked or faked because it's not an interface. Will discuss with John on Monday

Reply:
Realised that don't need a whole Competent auth object, just the ID so this mapper is no longer needed

Reply:
Just for information - We can fake the WeeeContext and set the data sets to return test lists, an example is in GetOrAddAatfDeliveryLocation_PopulatesCacheWithDatabaseValues

Reply:
Oh that's how you do it. I was trying to get it to return something from the query method I was calling. *Facepalm*